client/my-sites/domains/domain-management/name-servers/wpcom-nameservers-toggle.jsx

```diff
@@ -18,19 +18,17 @@ class NameserversToggle extends PureComponent {
 	};
 
 	render() {
+		const toggleTitle = this.props.translate( 'Use WordPress.com Name Servers' );
 		return (
 			<Card compact className="name-servers__dns">
-				<span className="name-servers__title">
-					{ this.props.translate( 'Use WordPress.com Name Servers' ) }
-				</span>
-
 				<form className="name-servers__toggle">
 					<ToggleControl
 						id="wp-nameservers"
 						name="wp-nameservers"
 						onChange={ this.handleToggle }
 						checked={ this.props.enabled }
 						value="active"
+						label={ toggleTitle }
```

Comment:
Now that `toggleTitle` is used only once, we don't need a variable; we can directly pass `this.props.translate( 'Use WordPress.com Name Servers' )` to the `label` prop.

Reply:
Haa...I think we have arrived... 🙌🏽